peachjam/templates/peachjam/layouts/document_detail.html

```diff
@@ -168,47 +168,48 @@ <h1 class="py-4">{{ document.title }}</h1>
                   {% endblock %}
                 </div>
 
-                {% block document-taxonomies %}
-                  {% if taxonomies %}
-                    {% include "peachjam/_taxonomies.html" %}
-                  {% endif %}
-                {% endblock %}
-
-                {% block document-relationships-content %}
-                  {% if n_relationships %}
-                    <div class="col-md">
-                      <h5>{% translate 'Related documents' %}</h5>
-
-                      <ul class="list-unstyled">
-                        {% if n_relationships > relationship_limit %}
-                          <li>
-                            <a href="#related-tab" data-component="ToggleTab">{% blocktrans %}{{ n_relationships }} related documents{% endblocktrans %}</a>
-                          </li>
-                        {% else %}
-                          {% for rel in relationships_as_subject %}
-                            {% if rel.object_work %}
-                              <li>
-                                {% translate rel.predicate.verb as verb %}
-                                {{ verb|capfirst }}
-                                <a href="{% url 'document_detail' frbr_uri=rel.object_work.frbr_uri|strip_first_character %}">{{ rel.object_work.title }}</a>
-                              </li>
-                            {% endif %}
-                          {% endfor %}
+                <div class="col-md">
```

Comment:
This undoes the functionality we added previously to allow the first column to take up all the space if there is no second column (which is pretty common). Adding this column should be guarded by sthing like `if taxonomies or n_relationships`.

Reply:
refactored with `{% if n_relationships or taxonomies %}` syntax